Paladin.cpp

```diff
@@ -1,7 +1,27 @@
 #include "Paladin.h"
+#include "Character.h"
```

Comment:
Paladin.h is already including Character.h
- [ ] revise.

Reply:
done.